tests/Testhelper/MyCustomSource.php

```diff
@@ -49,11 +49,11 @@ public function getRealPath(): string
     }
 
     /**
-     * @return int
+     * @return float
      */
-    public function getLastModifiedTime(): int
+    public function getLastModifiedTime(): float
     {
-        return $this->path === 'nocache' ? mt_rand() : 0;
+        return (float) ($this->path === 'nocache' ? mt_rand() : 0);
```

Comment:
I think this schould be int, or?

Reply:
https://github.com/SC-Networks/PHPTAL/blob/55761d54b81c0ba972ffb557d7a681f17cfeb994/src/PHPTAL.php#L1053
https://github.com/SC-Networks/PHPTAL/blob/55761d54b81c0ba972ffb557d7a681f17cfeb994/src/PHPTAL.php#L1067
https://github.com/SC-Networks/PHPTAL/blob/55761d54b81c0ba972ffb557d7a681f17cfeb994/src/PHPTAL.php#L916

Reply:
Changed to a saner way